tests/integration/kubernetes/run_kubernetes_tests.sh

```diff
@@ -128,16 +129,25 @@ fi
 
 ensure_yq
 
-info "Running tests with bats version: $(bats --version)"
+report_dir="${kubernetes_dir}/reports/$(date +'%F-%T')"
+mkdir -p "$report_dir"
+
+info "Running tests with bats version: $(bats --version). Save outputs to $report_dir"
 
 tests_fail=()
 for K8S_TEST_ENTRY in ${K8S_TEST_UNION[@]}
 do
 	info "$(kubectl get pods --all-namespaces 2>&1)"
 	info "Executing ${K8S_TEST_ENTRY}"
-	if ! bats --show-output-of-passing-tests "${K8S_TEST_ENTRY}"; then
+	# Output file will be prefixed with either "ok" or "not_ok" in case of
+	# success or fail, respectively.
+	out_file="${report_dir}/${K8S_TEST_ENTRY}.out"
+	if ! bats --show-output-of-passing-tests "${K8S_TEST_ENTRY}" | tee "$out_file"; then
 		tests_fail+=("${K8S_TEST_ENTRY}")
+		mv "${out_file}" "$(dirname "${out_file}")/not_ok-$(basename "${out_file}")"
```

Comment:
shouldn't `mv "${out_file}" "${report_dir}/not_ok-${K8S_TEST_ENTRY}.out"` do the same job?